src/core/main.cpp

```diff
@@ -236,7 +237,6 @@ int noInputFileError()
 	return usageError( "No input file specified" );
 }
 
-
```

Comment:
You may revert meaningless and unnecessary newline changes in some files.

Reply:
Done